dev/doc/univpoly.txt

```diff
@@ -12,7 +12,7 @@ type pinductive = inductive puniverses
 type pconstructor = constructor puniverses
 
 type constr = ...
-  | Const of puniversess
+  | Const of puniverses
```

Comment:
Shouldn't this be pconstant?

Reply:
@ejgallego ?

Reply:
This is the documentation of universes, not exactly a domain I'm comfortable with.

[I regret fixing the typo now XDD]

Reply:
So you mean we should merge as is, I guess?

Reply:
The mistake was there before the PR, I suggest we do a pass on the file on a separate PR if needed.